src/js/views/patients/patient/dashboard/dashboard_views.js

```diff
@@ -87,6 +90,11 @@ const ItemView = View.extend({
 
     this.showChildView('due', dueComponent);
   },
+  showAttachment() {
+    if (_.isEmpty(this.model.get('_forms'))) return;
```

Comment:
these `isEmpty` checks could all be `if (!this.model.getForm()))`   Try and keep the multiple forms thing to the entities layer